### ISSUE TITLE
Editor breaking cuisine data

### ISSUE THREAD

**Issue by matheusgomesms** [organicmaps/organicmaps#9404]
I was updating `cuisine` values on OM and it deleted OSM data. Case:

1 - restaurant had a sole `cuisine=nepalese` on OSM
2 - since it does not have an equivalent on OM, it did not show up on Place Page/Editor etc
3 - also, since I did not know it was tagged before, I went to add `cuisine=asian` (the closest one I could figure out from OM editor)
4 - in OSM, the `nepalese` value was replaced to asian

(check v7 and v8 here): https://www.openstreetmap.org/node/2943227934/history

Proposed workflow: if someone adds a `cuisine` value on editor, just ADD the new information, do not replace it on OSM.

Chiming in @map-per, since their recent work on Editor.

**Comment by matheusgomesms** [comment]
Possibly related to #6292

**Comment by RedAuburn** [comment]
this would be fixed by https://github.com/organicmaps/organicmaps/pull/9242

**Comment by map-per** [comment]
No, unfortunately #9242 doesn't solve this problem because all cuisines are stored in one OSM tag. This needs an extra fix. But the data stored in the EditJournal can be used to also fix this problem.
